creme/activities/views/calendar.py

```diff
@@ -52,15 +46,34 @@
 from ..models import Calendar
 from ..utils import check_activity_collisions, get_last_day_of_a_month
 
+# from creme.creme_core.utils.dates import make_aware_dt
 logger = logging.getLogger(__name__)
 Activity = get_activity_model()
 
 
-def _js_timestamp_to_datetime(timestamp):
+def fromRFC3339(timestamp):
     # "@raise ValueError."
     # JS gives us milliseconds
     # return make_aware_dt(datetime.fromtimestamp(float(timestamp) / 1000))
-    return make_aware(datetime.fromtimestamp(float(timestamp) / 1000))
+    if timestamp is None:
+        return None
+
+    res = datetime.fromisoformat(timestamp[:-2] if timestamp[-1] == 'Z' else timestamp)
+    return res.replace(tzinfo=get_current_timezone())
+    # return make_aware(datetime.fromtimestamp(float(timestamp) / 1000))
+
+
+def toRFC3339(value):
```

Comment:
I presume you can add some static typing here because we love that kind of things.